compleet-client/src/ui/buffer.rs

```diff
@@ -6,19 +6,19 @@ use crate::bindings::api;
 
 #[derive(Debug, PartialEq)]
 pub struct Buffer {
-    pub number: u32,
+    pub bufnr: u16,
 }
```

Comment:
why renaming?

Reply:
I figured 'bufnr' would be more descriptive instead of 'self.number' for example. I can change it back if you'd like.

Reply:
But that `self` is already a `Buffer`. The `buf` part of `bufnr` seems redundant imho.